plugins/org.eclipse.n4js.transpiler/src/org/eclipse/n4js/transpiler/sourcemap/Base64VLQ.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2018 Jens von Pilgrim.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Jens von Pilgrim - Initial API and implementation
```

Comment:
I understand that this will lead to other contributions with authors mentioned in the file header, also from the IDE team. Until now I thought we only add authors when existing code was copied from somewhere else. The new rule would be that author information can be added by anybody iff the file is newly created on the weekend day, right?